docs/supported.md

```diff
@@ -471,24 +504,26 @@ $\textcolor{#228B22}{F=ma}$ `\textcolor{#228B22}{F=ma}`<br>
 $\colorbox{aqua}{A}$ `\colorbox{aqua}{A}`<br>
 $\fcolorbox{red}{aqua}{A}$ `\fcolorbox{red}{aqua}{A}`
 
-For color definition, KaTeX color functions will accept the standard HTML [predefined color names](https://www.w3schools.com/colors/colors-names.asp). They will also accept an RGB argument in CSS hexa­decimal style.
+For color definition, KaTeX color functions will accept the standard HTML [predefined color names](https://www.w3schools.com/colors/colors_names.asp). They will also accept an RGB argument in CSS hexa­decimal style.
 
 **Font**
 
 ||||
-|:----------------------------------|:----------------------------------|:-----
-|$\mathrm{AB}$ `\mathrm{AB}`  |$\mathbf{AB}$ `\mathbf{AB}`  |$\mathit{AB}$ `\mathit{AB}`
-|$\textrm{AB}$ `\textrm{AB}`  |$\textbf{AB}$ `\textbf{AB}`  |$\textit{AB}$ `\textit{AB}`
-|$\rm AB$ `\rm AB`|$\bf AB$ `\bf AB`|$\it AB$ `\it AB`
-|$\textnormal{AB}$ `\textnormal{AB}`|$\bold{AB}$ `\bold{AB}`|$\Bbb{AB}$ `\Bbb{AB}`
-|$\text{AB}$ `\text{AB}`|$\boldsymbol{AB}$ `\boldsymbol{AB}`|$\mathbb{AB}$ `\mathbb{AB}`
-|$\mathsf{AB}$ `\mathsf{AB}`  |$\bm{AB}$ `\bm{AB}` |$\frak{AB}$ `\frak{AB}`
-|$\textsf{AB}$ `\textsf{AB}`  |$\mathtt{AB}$ `\mathtt{AB}`  |$\mathfrak{AB}$ `\mathfrak{AB}`
-|$\sf AB$ `\sf AB`|$\texttt{AB}$ `\texttt{AB}`  |$\mathcal{AB}$ `\mathcal{AB}`
-|  |$\tt AB$ `\tt AB`|$\mathscr{AB}$ `\mathscr{AB}`
+|:------------------------------|:------------------------------|:-----
+|$\mathrm{Ab0}$ `\mathrm{Ab0}`  |$\mathbf{Ab0}$ `\mathbf{Ab0}`  |$\mathit{Ab}$ `\mathit{Ab}`
+|$\textrm{Ab0}$ `\textrm{Ab0}`  |$\textbf{Ab0}$ `\textbf{Ab0}`  |$\textit{Ab}$ `\textit{Ab}`
+|$\rm Ab0$ `\rm Ab0`            |$\bf Ab0$ `\bf Ab0`            |$\it Ab$ `\it Ab`
+|$\textnormal{Ab0}$ `\textnormal{Ab0}`|$\bold{Ab0}$ `\bold{Ab0}`|$\Bbb{AB}$ `\Bbb{AB}`
+|$\text{Ab0}$ `\text{Ab0}`      |$\boldsymbol{Ab}$ `\boldsymbol{Ab}`|$\mathbb{AB}$ `\mathbb{AB}`
+|$\mathsf{Ab0}$ `\mathsf{Ab0}`  |$\bm{Ab}$ `\bm{Ab}`            |$\frak{Ab0}$ `\frak{Ab0}`
```

Comment:
Is there a reason that a few of these don't have 0s added?

Reply:
> Is there a reason that a few of these don't have 0s added?

Yes, those fonts don't contain a "0" glyph.

Reply:
`\boldsymbol` and `\bm do render a 0, but it's not bold-*italic*.  It's just bold.